textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/FuzzyTokenizer.java

```diff
@@ -0,0 +1,33 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ * Class to take in a query and with a dictionary of words split it into possible set of queries
```

Comment:
In fact, since the purpose for PR is to design the interface and add test cases, we should NOT include any implementation-related logic.  So I suggest you remove all these classes related to the implementation.  Add them in a separate PR in the future.